language/transaction-builder/src/lib.rs

```diff
@@ -134,20 +134,15 @@ encode_txn_script! {
 pub fn encode_transfer_with_metadata_script(
     type_: TypeTag,
     recipient: &AccountAddress,
-    auth_key_prefix: Vec<u8>,
```

Comment:
Mega nit: now that we don't have the validation of the auth_key_prefix, this can be written using the `encode_txn_script!` macro.

Reply:
Good call; fixed!